docs/external-runtime-deps.md

```diff
@@ -191,6 +191,21 @@ are not detected.
 External `id` will be executed as a fallback if local user lookup fails, in case
 NSS is used.
 
+#### sh
+
+`sh` is executed by k0s when CPLB is used with `VirtualServers`. CPLB with userspace
+proxy does not require it.
+
+#### sed
+
+`sed` is executed by k0s when CPLB is used with `VirtualServers`. CPLB with userspace
+proxy does not require it.
+
+#### kill
+
+`kill` is executed by k0s when CPLB is used with `VirtualServers`. CPLB with userspace
+proxy does not require it.
```

Comment:
In the way kill is currently used, it may very well be that it's coming as a shell builtin. All the shells have this. I checked at least bash/dash/ash/zsh/ksh. So maybe we can join this into the sh section, and mention that if the shell doesn't provide a kill builtin (which is very unlikely I'd say), then you need the binary from `util-linux`.